arranger/Chart.yaml

```diff
@@ -14,4 +28,4 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.3.0
+version: 0.3.1
```

Comment:
is this really a patch version? a lot of changes here.

Reply:
yes, since all changes are non-breaking... this chart works just fine with Arranger v2